stdlib/bz2.pyi

```diff
@@ -82,7 +82,7 @@ def open(
 ) -> TextIO: ...
 
 class BZ2File(BaseStream, IO[bytes]):
-    def __enter__(self: _T) -> _T: ...
+    def __enter__(self: Self) -> Self: ...
```

Comment:
This method doesn't actually exist at runtime; I guess it is inherited from an IO-related base class.

Reply:
You are correct, I think it is IOBase, but I would need to check.